core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -45,6 +45,13 @@
 -export([delete_system_media_references/0]).
 -export([migrate_system/0]).
 
+-export([binding/1]).
+
+binding(migrate) -> <<"maintenance.migrate">>;
+binding(refresh_webhooks) -> <<"maintenance.refresh_webhooks">>;
+binding(refresh_stepswitch) -> <<"maintenance.refresh_stepswitch">>;
+binding(refresh_account_db) -> <<"maintenance.refresh_account_db">>.
```

Comment:
I think binding for account DBs should be its own function: `refresh_account_binding(AccountDb) -> <<"maintenance.refresh.account.", AccountDb/binary>>.`. Then I can bind for `refresh_account_binding(<<"*">>)` in the maintenance module of my choice.